pkg/plugins/policies/meshhttproute/plugin/v1alpha1/listeners.go

```diff
@@ -139,6 +139,10 @@ func prepareRoutes(
 ) []api.Route {
 	// policy matching for real MeshService is not yet ready
 	conf := rules.ComputeConf[api.PolicyDefault](toRules, core_rules.MeshService(svc.ServiceName))
+	switch svc.BackendRef.Kind {
+	case common_api.MeshExternalService:
+		conf = rules.ComputeConf[api.PolicyDefault](toRules, core_rules.MeshExternalService(svc.ServiceName))
```

Comment:
do we need this specific `MeshExternalService` subset? It basicaly is the same as MeshService. Do we expect to match on anything else than service tag in the future?

Reply:
@lukidzi - WDYT? I think it was just safer so that if MeshService changes then we won't change and we can change without the need to change MS.

Reply:
I think it's a good way to distinguish between 2 different types. We shouldn't mix meshservice and meshexternalservice